<launch>

	<node pkg="cat_robotcam_calibration" type="robot_camera_calibration" name="robot_camera_calibration" output="screen"/> 

	<node pkg="cat_robotcam_calibration" type="transformConverter" name="transformConverter" output="screen"/> 
	<node pkg="cat_move_to_target" type="gettagpose_service" name="gettagpose_service" output="screen"/> 

</launch>
